### PR TITLE
[PL-51486] PL subscriptions - Cloud credits

### PR DESCRIPTION
**Pull request by brian-f-harness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: PL subscriptions - Cloud credits new section, CI subscriptions - update "build credits" to "cloud credits"
* Jira/GitHub Issue numbers (if any): PL-51486
* Preview links/images (Internal contributors only): 
   - [PL subscriptions - Cloud credits](https://667c1730312e1d12c8bf1bea--harness-developer.netlify.app/docs/platform/get-started/subscriptions-licenses/subscriptions#cloud-credits)

   - [CI subscriptions and licenses](https://667f714665c136b55a1cd290--harness-developer.netlify.app/docs/continuous-integration/get-started/ci-subscription-mgmt)

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [x] Code owner review.
- [x] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/platform/shared/subscription-cloud-credits.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+**Credits Breakdown** enables you to view your credit usage by organization, project, or resource class. You can also use the date filters to view by a **Recommended** range, **Relative dates**, **Calendar months**, or a selected custom range.
+
+#### Cloud Credits Allocation
+
+**Cloud Credits Allocation** enables you to view the following details for your credit allocation:
+
+- **Start Date:** The date on which a block of credits were applied to your account.
+- **Expiry Date:** The date on which a block of credits expires.
+- **Type:** The credit type, Free, Team, or Enterprise.
```
Comment (douglas-j-bothwell) [inline docs/platform/shared/subscription-cloud-credits.md:30]:
```suggestion
- **Type:** The credit type: Free, Team, or Enterprise.
```

**Review by douglas-j-bothwell — APPROVED** [review]
One punctuation tweak, otherwise LGTM

**Review by batraprashant — APPROVED** [review]

**Comment by apardeshi-harness** [comment]
@brian-f-harness 

- The "Cloud Credits Allocation" is now just "Credits Allocation"

- Can we update the print shot with the most recent one? I have shared my qa account details with you

- Can we change this "Available Credits: Current balance of credits remaining to use." to "Available Credits: Current balance of active credits remaining to use."

- "Next Expiring Date" is not "Next Expiry Date"

File: docs/platform/shared/subscription-cloud-credits.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+   - The active credit period is the time between the **Start Date** and **Expiry Date** for your oldest unexpired credits.
+   - For free accounts, the credit balance resets at the end of every calendar month. Your **Start Date** is the first of the month. Your **Expiry Date** is the end of the month.
+   - If you purchased additional credits that haven't yet expired, the **Start Date** for **Used Credits** is the date that you purchased the additional credits, and the active credit period covers the lifetime of your purchased credits. For example, if you purchase credits in January that last for six months, **Used Credits** tracks credit usage from January through July, when your purchased credits expire. Therefore, **Used Credits** could include credits with various expiration dates.
+- **Available:** The number of credits available to use.
+- **Status:** The credit status, **Active** or **Expired**.
+
+For more information about Harness Cloud billing and build credit consumption, go to [Subscriptions and licenses](/docs/continuous-integration/get-started/ci-subscription-mgmt.md#harness-cloud-billing-and-build-credits).
```
Comment (brian-f-harness) [inline docs/platform/shared/subscription-cloud-credits.md:39]:
```suggestion
For more information about Harness Cloud billing and cloud credit consumption, go to [Subscriptions and licenses](/docs/continuous-integration/get-started/ci-subscription-mgmt.md#harness-cloud-billing-and-build-credits).
```

File: docs/platform/shared/subscription-cloud-credits.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+![](./static/cloud-credits.png)
+
+#### Credits & Usage
+
+**Credits & Usage** enables you to view the following credit information:
+
+- **Available Credits:** Current balance of active credits remaining to use.
+- **Month-to-date Credit Usage:** Credits consumed month-to-date.
+- **Next Expiry Date:** The next upcoming credit expiration date.
+- **Credit Overage:** Current amount of credit overage. As a courtesy, Harness notifies you when you are running low on build credits. For more information, go to [Credit allowance by plan tier](/docs/continuous-integration/get-started/ci-subscription-mgmt/#credit-allowance-by-plan-tier) and [Credit overages (overuse)](/docs/continuous-integration/get-started/ci-subscription-mgmt#credit-overages-overuse).
```
Comment (brian-f-harness) [inline docs/platform/shared/subscription-cloud-credits.md:18]:
```suggestion
- **Credit Overage:** Current amount of credit overage. As a courtesy, Harness notifies you when you are running low on cloud credits. For more information, go to [Credit allowance by plan tier](/docs/continuous-integration/get-started/ci-subscription-mgmt/#credit-allowance-by-plan-tier) and [Credit overages (overuse)](/docs/continuous-integration/get-started/ci-subscription-mgmt#credit-overages-overuse).
```

**Review by batraprashant — APPROVED** [review]

**Pull request merged**